src/commands/appSettings/IAzureWebJobsStorageWizardContext.ts

```diff
@@ -3,9 +3,11 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { StorageAccount } from "@azure/arm-storage";
 import { IActionContext, ISubscriptionContext } from "@microsoft/vscode-azext-utils";
 
 export interface IAzureWebJobsStorageWizardContext extends IActionContext, Partial<ISubscriptionContext> {
     projectPath: string;
+    storageAccount?: StorageAccount;
```

Comment:
Does this actually get set anywhere? I'd assume not since you just added it to the context definition 😅

Reply:
I was also thinking that when I originally went to write this code I found the storage account is set in StorageAccountListStep.js which I couldn't find a way to access that data, but I randomly went to debug my code it does seem to set the storage account data correctly as shown in the image below. 

![image](https://user-images.githubusercontent.com/59709511/193859671-df8b7355-d4b6-45f8-9df5-b414212a4117.png)

Reply:
Oh right, in that case, what you probably should do is just extend `IStorageAccountWizardContext` from the `vscode-azext-azureutils` package since that's where it is actually setting the `storageAccount` property.